### DISCUSSION TITLE
This discussion is incorrect

### DISCUSSION THREAD

**Discussion by jbteves** [ericearl/fancy_github#12] [Q&A]
Hi Eric, could you please tell us the most incorrect fact you know?

**Comment by ericearl** [comment]
@jbteves I am worried that providing an answer would go against the sentiment of the question, but I'll try to answer the question under two likely incorrect assumptions instead:

### 1. Assume you mean "the most incorrect fact you think people believe is true"

I think a common misconception of this nature is dogs don't have emotional feelings.  It may be true that cats don't actually care about you, but they let you live with them anyway because you have the thumbs to open the cans.

### 2. Assume you mean "the thing you thought was a fact the longest that turned out to be wrong"

For the longest time as an adolescent I thought no two song titles could be the same.
